app/models/geometric-property.js

```diff
@@ -50,6 +51,51 @@ export const GEOMETRY_TYPES = [
   'rezoningArea',
 ];
 
+// returns true or false based on whether the change to the geometric
+// property was "meaningful"
```

Comment:
might be helpful to include a description of the different cases that constitute a meaningful chanfe